Jint.Tests/Runtime/Domain/JsUuid.cs

```diff
@@ -3,25 +3,38 @@
 
 namespace Jint.Tests.Runtime.Domain
 {
-    public sealed class JsUuid : JsValue, IEquatable<JsUuid>
+public sealed class JsUuid : IEquatable<JsUuid>
+{
+    internal readonly Guid _value;
+
+    public static readonly JsUuid Empty = new JsUuid(Guid.Empty);
+
+    public static JsUuid Parse(JsValue value)
     {
-        internal readonly Guid _value;
-        public static readonly JsUuid Empty = new JsUuid(Guid.Empty);
+        return value switch
+        {
+            { } js when Guid.TryParse(js.AsString(), out var res) => new JsUuid(res),
```

Comment:
```suggestion
            IObjectWrapper ow when ow.Target is JsUuid res => res,
            { } js when Guid.TryParse(js.AsString(), out var res) => new JsUuid(res),
```

can already be a JsUuid. `Uuid.parse(Uuid.Empty)`